paparazzi-gradle-plugin/src/test/java/app/cash/paparazzi/gradle/PaparazziPluginTest.kt

```diff
@@ -205,6 +205,37 @@ class PaparazziPluginTest {
     assertThat(snapshotsDir.exists()).isTrue()
   }
 
+  @Test
+  fun junitReportSnapshot() {
+    val fixtureRoot = File("src/test/projects/junit-report-snapshots")
+    val testFile = File(fixtureRoot, "src/test/java/app/cash/paparazzi/plugin/test/LaunchViewTest.kt")
+    val failingTestFile = File(fixtureRoot, "src/test/java/app/cash/paparazzi/plugin/test/LaunchViewTest.kt.tmp")
```

Comment:
Instead of having a separate tmp file, should the test itself just be wrong to the golden snapshot?

Reply:
ya, that might make things simpler